README.md

```diff
@@ -0,0 +1,14 @@
+```
+Process a Markdown file, execute commands within 
+and print file contents with executed commands' output.
+
+Usage:
+    markdown-command [options] FILE
+
+The pattern for commands to be executed is:
+    [$ command --with-options -and arguments]
+
+Options:
+    --help     Display this message.
+    --version  Display version information.
+```
```

Comment:
I don't think it's good that we have everything inside code block. I would change that a little bit to:

````suggestion
Process a Markdown file, execute commands within 
and print file contents with executed commands' output.

```
Usage:
    markdown-command [options] FILE

The pattern for commands to be executed is:
    [$ command --with-options -and arguments]

Options:
    --help     Display this message.
    --version  Display version information.
```
````

Otherwise, it will look confusing if you put everything in the code block and not explanation whatsoever. Does that make sense?